packages/fluentui/accessibility/src/behaviors/Carousel/carouselBehavior.ts

```diff
@@ -80,6 +90,8 @@ export type CarouselBehaviorProps = {
   /** Element type. */
   navigation: Object | Object[];
   ariaLiveOn: boolean;
+  paddleNextHidden: boolean;
+  paddlePreviousHidden: boolean;
```

Comment:
making them optional should stop unit test from screaming :)

Reply:
I was thinking as well about it, if props should be optional or not... but the info about visibility should be there, because paddle should be either visible or not... if I am thinking correctly... :) so I would rather let them not optional...